base/.env

```diff
@@ -11,4 +11,7 @@ SSCAN_VERSION=R2-11-5
 RECCASTER_VERSION=1.5
 IPAC_VERSION=2.16
 AREA_DETECTOR_VERSION=R3-12-1
+NDSSCPIMEGA_VERSION=1.0.0
 MOTOR_VERSION=R7-3-1
+
+LIBSSCPIMEGA_VERSION=main
```

Comment:
This is an unfortunate configuration. I'll see if we can have a tag available for downloading with `download_github_module`.

Reply:
Switching this PR to draft until I fix this for non-rolling module versioning.